packages/cli/src/ceramic-daemon.ts

```diff
@@ -164,15 +152,25 @@ export class CeramicDaemon {
     const ceramicConfig = makeCeramicConfig(opts)
 
     const ipfs = await buildIpfsConnection(
-      opts.network,
+      opts.ipfs.mode,
+      opts.network?.name,
       ceramicConfig.loggerProvider.getDiagnosticsLogger(),
-      opts.ipfsHost
+      opts.ipfs?.host
     )
 
     const [modules, params] = Ceramic._processConfig(ipfs, ceramicConfig)
-
-    if (opts.s3StateStoreBucket) {
-      const s3StateStore = new S3StateStore(opts.s3StateStoreBucket)
+    modules.loggerProvider
```

Comment:
Added this log with the daemon config, and removed the log from within Ceramic itself that logs the config.  This seems more useful, and doesn't require any special work to massage complex objects into a format where they can be logged.